reporter/html.js

```diff
@@ -460,6 +460,18 @@ function storeFixture() {
 	}
 }
 
+function appendFilteredTest() {
+	var testId = QUnit.config.testId;
+	if ( testId && testId.length > 0 ) {
+		return "<div id='qunit-filteredTest'>Running filtered test: " + testId.join(", ") +
+			" <a id='qunit-clearFilter' href='" +
+			setUrl({ filter: undefined, module: undefined, testId: undefined }) +
+			"'>" + "Clear filter" + "</a></div>";
+	} else {
```

Comment:
We added this because if the function doesn't return anything, the div will show up as "undefined".